core/src/main/java/io/ddf/datasource/DataFormat.java

```diff
@@ -0,0 +1,15 @@
+package io.ddf.datasource;
+
+
+/**
+ */
+
+public enum DataFormat {
+  UNDEF(-1), SQL(0), CSV(1), TSV(2), JSON(3), PARQUET(4);
+
+  private int value;
+
+  private DataFormat(int value) {
+    this.value = value;
+  }
```

Comment:
We'd need this to be able to convert int to a DataFormat:

    public static DataFormat fromInt(int x) {
      switch(x) {
        case 0:
          return SQL;
        case 1:
          return CSV;
        case 2:
          return TSV;
        case 3:
          return JSON;
        case 4:
          return PQT;
        default:
          return UNDEF;
      }
    }